packages/website/README.md

```diff
@@ -0,0 +1,81 @@
+# Cadl Website using 11Ty Static Website Generator
+
+This repo contains the code for static web generator for Cadl project using Eleventy(11ty) static web generator. To get started with 11Ty visit https://www.11ty.dev.
+
+Support libraries that enable the demo features are in the `packages/` directory. The demo code itself is in the `website/` directory.
+
+## Instructions for editing or contributing to this website
+
+### Prerequisites
+
+#### Install Node & NPM
+
+Downlaod and install from https://nodejs.org.
+
+#### Install Eleventy (11Ty) using NPM.
+
+```bash
+> npm i --save-dev @11ty/eleventy

```

Comment:
Don't need that step, was just needed for you the first time but now it is part of the pacakge.json